devops_sccs/plugins/bitbucketcloud.py

```diff
@@ -334,6 +335,11 @@ async def trigger_continuous_deployment(self, session, repository, environment,
 
                 deploy_branch = repo.refs().branches.by_name(branch)
                 await deploy_branch.get()
+                #If the branch already exist , we should remove it.
+                try:
+                      deploy_branch.delete()
```

Comment:
this is a coroutine so please use await in front

see https://github.com/croixbleueqc/python-aiobitbucket#delete-a-branch-not-the-main-one